examples/repository/DBStructure.md

```diff
@@ -0,0 +1,15 @@
+# About the MongoDB for this Example
```

Comment:
Thank you for documenting the use case here. However, the .md file is not necessary. We should try to keep the example use case document in the code. For example, in the example_test.go file we could do something like:

```go
// The Example DB used in this is a MongoDB Cluster and has the following:
// - One database: "GidariExample"
// - Two collections: "ExampleTable", "AnotherExampleTable"
```